### DISCUSSION TITLE
ansible upgrade plans

### DISCUSSION THREAD

**Discussion by domrim** [Homebrew/discussions#107] [Everyday usage] [answered]
hi everybody,

as some of you may have noticed the ansible project has been restructured and splited in two parts,
ansible and ansible-base as described in their [blog](https://www.ansible.com/blog/thoughts-on-restructuring-the-ansible-project).

The ansible package in pypi contains only the additionall collections (these contain the "modules" ) and ansible-base provides the core functionality. So ansible depends now on ansible-base. My question is how the homebrew project should handle this change.

Should we still ship only one ansible-package which contains everything or do the same spliting like the ansible project?

**Comment by SMillerDev** [comment, marked as answer]
Unless both projects work independently it should just be one formula.

**Reply by domrim** [reply]
okay, is there a "standart" way for python projects to install a second package aside of the "main" package and link its binaries?
i read through https://docs.brew.sh/Python-for-Formula-Authors but dont have a good idea to solve this problem.

**Reply by SMillerDev** [reply]
All python formulae are installed in their own virtual environment so they don't and can't interact.
